keccak256/src/permutation/rho_helpers.rs

```diff
@@ -124,8 +117,6 @@ pub struct RhoLane {
     input: BigUint,
     // base 9
     pub output: BigUint,
-    // output in binary, useful to check against plain rho output
-    output_b2: u64,
```

Comment:
Can we make `output_b2`, `chunk_idx`, and `step` public and keep them? They might come in handy when debugging Rho in the future.

Reply:
So I removed them since we should not need to debug Rho anymore in the future.
As basically, any changes now would not touch the permutation.

But I can roll it back and add an exception for `unused_code`. Note that making them `pub` will not remove the err away IIRC.

Reply:
Your call! :smile: